templates/bot/js/command-and-response/README.md

```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
 | `src/helloworldCommandHandler.js` | The business logic to handle a command |
 | `src/cardModels.js` | The default Adaptive Card data model |
 
-The following files implement the core command and response on the Bot Framework. You generally will not need to customize these files.
-
-| File / Folder | Contents |
-| - | - |
-| `src/internal/initialize.js` | Application initialization and bot message handling |
-
-The following files are project-related files. You generally will not need to customize these files.
-
-| File / Folder | Contents |
-| - | - |
-| `.gitignore` | Git ignore file |
-| `package.json` | NPM package file |
-
-# Customize your application
-
-By default a single command is generated that sends the `helloworldCommand.json` Adaptive Card when a user types `helloWorld` in the private message chat with the bot.
+## Extend the command bot template with more commands and responses
 
-This section outlines some customization you can do to adopt the application for your needs.
+Follow the steps below to add more commands and responses to extend the command bot:
 
-## Customize the trigger pattern
+1. [Step 1: Add a command definition in manifest](#Step-1-Add-a-command-definition-in-manifest)
+1. [Step 2: Respond with an Adaptive Card](#Step-2-Respond-with-an-Adaptive-Card)
+1. [Step 3: Handle the command](#Step-3-Handle-the-command)
+1. [Step 4: Register the new command](#Step-4-Register-the-new-command)
 
-The default pattern to trigger a command is through a defined keyword. But often times you would want to collect and process additional information retrieved after the trigger keyword. In addition to keyword match, you could also define your trigger pattern with [regular expressions](https://regex101.com/) and match against `message.text` with more controls.
+### Step 1: Add a command definition in manifest
 
-When using regular expressions, any capture group can be found in `message.matches`. Below is an example that uses regular expression to capture strings after `reboot`, for example if user inputs `reboot myMachine`, `message.matches[1]` will capture `myMachine`:
+You can edit the manifest template file `templates\appPackage\manifest.template.json` to include definitions of a command with its title and description in the `commands` array:
```

Comment:
a command -> a `doSomething` command?
The `helloWorld` command is generated by default, while `doSomething` is added for customization.

Reply:
Done